docs/usage/cli.md

```diff
@@ -1,535 +1,724 @@
+  --input_params "param1 INT, param2 INT" \
+  --data_type INT \
+  --def "return param1 + param2" \
+  --comment "Sum Function"
 ```
 
-### 5.4 Invoke a Function
+### Invoke a Function
+
+You can invoke a function stored in Unity Catalog using:
 
 ```sh
-bin/uc function call --full_name <catalog>.<schema>.<function_name> --input_params <input_params>
+bin/uc function call \
+  --full_name <catalog>.<schema>.<function_name> \ # (1)
+  --input_params <input_params> # (2)
 ```
 
-- `full_name`: The full name of the function, which is a concatenation of the catalog name, schema name, and function
-    name separated by dots (e.g., `catalog_name.schema_name.function_name`).
-- `input_params` : The value of input parameters to the function separated by a comma (e.g., `"param1,param2"`).
-
-This is an experimental feature and only supported for python functions that take in primitive types as input
-parameters. It runs the functions using the python engine script at `etc/data/function/python_engine.py`.
+1. `full_name`: The full name of the function, which is a concatenation of the catalog name, schema name, and function name separated by dots (e.g., `catalog_name.schema_name.function_name`).
+2. `input_params` : The value of input parameters to the function separated by a comma (e.g., `"param1,param2"`).
 
-Example:
+This is an experimental feature and only supported for Python functions that take in primitive types as input
+parameters. It runs the functions using the Python engine script at `etc/data/function/python_engine.py`.
 
-- Invoke a python sum function that takes two integer inputs:
+Here's an example that invokes a Python sum function that takes two integer inputs:
 
 ```sh
-bin/uc function call --full_name my_catalog.my_schema.my_function --input_params "1,2"
+bin/uc function call \
+  --full_name my_catalog.my_schema.my_function \
+  --input_params "1,2"
 ```
 
-### 5.5 Delete a Function
+### Delete a Function
+
+You can delete a function using:
 
 ```sh
-bin/uc function delete --full_name <catalog>.<schema>.<function_name>
+bin/uc function delete \
+  --full_name <full_name> # (1)
 ```
 
-- `catalog` : The name of the catalog.
-- `schema` : The name of the schema.
-- `function_name` : The name of the function.
+1. `full_name`: The full name of the function, which is a concatenation of the catalog name, schema name, and function
+   name separated by dots (e.g., `catalog_name.schema_name.function_name`).
+
+## Registered model and model version management
 
-## 6. Registered model and model version management
+You can use Unity Catalog with MLflow to govern and access your ML and AI models. Read more
 
 Please refer to [MLflow documentation](https://mlflow.org/docs/latest/index.html) to learn how to use MLflow to create,
 register, update, use, and delete registered models and model versions.
 
-## 7. CLI Server Configuration
+You can list the registered models in your UC namespace using:
+
+```bash title="List registered models"
```

Comment:
Use `sh` instead to stay consistent. Also applies to the other code snippets below.